### PR TITLE
!caesar phrase to cypher

### LINKED ISSUE #72: Create a new parser for Caesar Cypher
A new parser could be created for using Caesar Cypher (ROT13 Shift).
The new parser could be triggered by the use of the `!caesar` token followed by the phrase to encrypt. An example case is shown below:
```text
User> !caesar hello world!
Bot > uryyb jbeyq!
```

### PR DESCRIPTION
**Pull request by sisco0**
The current PR resolves #72 by creating a new `!caesar` command that could be used to cypher texts using the known ROT13 algorithm.

### CODE REVIEW

**Comment by sisco0** [comment]
Unfortunately, I could not find a way to test it _online_. Only offline tests were done by defining a function under a Javascript sandbox (NodeJs).

Could you kindly set a review on the function considering that this is my first pull request for the project?

Thank you in advance!

File: Parsers/CaesarCypher.js [hunk 1]
```diff
@@ -0,0 +1,33 @@
+/*
+activation_example:!caesar your sentence
+regex:!caesar
+flags:g
+*/
+
+var where = 0;
+
+//Determine if the input was '!caesar'
+where = current.text.indexOf('!caesar') + 8;
```
Comment (earlduque) [inline Parsers/CaesarCypher.js:10]: works but unnecessary since there is only one activation phrase

File: Parsers/CaesarCypher.js [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+//Send the Slack message
+var sendIt = new x_snc_slackerbot.Slacker().send_chat(current, msg, false);
+
+function caesarString(aString) {
+    var last = aString.length;
+    var result = new Array(aString.length);
+    for (var i = 0; i <= last; i++) {
+        var c = aString.charCodeAt(i);
+        if (c >= 97 && c <= 122) {
+            c += 13; // ROT13 is used for Caesar cypher (shift)
```
Comment (earlduque) [inline Parsers/CaesarCypher.js:25]: maybe could have defaulted to 13 but allowed for user to determine how much the alphabet shifts?

**Review by earlduque — APPROVED** [review]
Some things can be cleaned or inline comments could have been updated but the code works and we can always update it further in the future. approved

**Comment by earlduque** [comment]
Hey @sisco0 thanks for your contribution, fyi i changed your PR description from
`The current PR solves issue https://github.com/ServiceNowDevProgram/SlackerBot/issues/72`
to simply
`The current PR resolves #72`
When you do `keyword #XX` it will auto close the related issue for the maintainer when the merge is committed: https://docs.github.com/en/issues/tracking-your-work-with-issues/linking-a-pull-request-to-an-issue

**Pull request merged**